dataprofiler/labelers/base_data_labeler.py

```diff
@@ -17,7 +18,11 @@
 from . import data_processing
 from .base_model import BaseModel
 
-default_labeler_dir = pkg_resources.resource_filename("resources", "labelers")
+if sys.version_info >= (3, 9):
+    default_labeler_dir = str(importlib.resources.files("resources")) + "/labelers"
```

Comment:
I saw the prev comment, but does:
`default_labeler_dir = importlib.resources.files("resources").joinpath("labelers")` 
Not work here now with the new changes?

Reply:
I'm concerned about the brittleness of ` + "/labelers"`

Reply:
This leads me to believe it should work in 3.9:
https://docs.python.org/3/library/importlib.resources.abc.html#importlib.resources.abc.Traversable

if not, we could req the version for `joinpath` to be 3.10 and higher

Reply:
Using `default_labeler_dir = importlib.resources.files("resources").joinpath("labelers")`
returns this error: `No overload variant of "join" matches argument types "Traversable", "str"`

The Python version I am using locally is 3.10

Reply:
This is a mypy error, so I used this: `str(importlib.resources.files("resources")) + "/labelers"` in order to avoid triggering the mypy error of: `No overload variant of "join" matches argument types "Traversable", "str"`.

I will try to come up with a different solution